src/ci/bin/testCentaurTes.sh

```diff
@@ -60,7 +60,7 @@ startup_funnel
 
 # Limiting Centaur to 4 threads here in order to try to prevent exceeding the OS's max open files limit: BA-6153
```

Comment:
Do we want to change this comment to say `8` or remove it if it's no longer relevant?